engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/definition/WithExplicitPossibleProducedVariableTypes.scala

```diff
@@ -0,0 +1,7 @@
+package pl.touk.nussknacker.engine.definition
+
+import pl.touk.nussknacker.engine.api.typed.typing.TypedClass
+
+trait WithExplicitPossibleProducedVariableTypes {
+  def possibleVariableClasses: Set[TypedClass]
```

Comment:
let's rename it somehow. These are not necessarily exact variable classes. Maybe sth along the lines of: 
`trait DeclaringExplicitTypesToExtract {
   def typesToExtract: Set[TypedClass]
}
`?

Reply:
Renamed.